coralproject/talk
# Added joi plugin validation

## What does this PR do?

- Adds support for plugin hooks to get validated before they are loaded to avoid runtime errors with plugins

## How do I test this PR?

- Try with an invalid plugin config, notice it won't start
- Try with a valid plugin config, notice it will start!
